### ISSUE TITLE
[FEATURE REQUEST] Add release

### ISSUE THREAD

**Issue by darigovresearch** [czM1K3/keychain-generator#17]
Would be nice to have a V1.0.0 release in GitHub to make it clearer to see when there are major & minor releases.

Instructions on how to do this can be seen [here](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)

Happy to do this if you wish just need to be given admin or maintainer rights if you're happy to do so

**Comment by darigovresearch** [comment]
@czM1K3 did you want to do this or are you happy with giving us admin/maintainer rights on this repo to add this?

**Comment by czM1K3** [comment]
I'm no sure about doing 1.0.0 release, because it is not production ready. There are a lot's of bugs and it's not ready for that. But i can create release to something like 0.0.1. What do you think?

**Comment by darigovresearch** [comment]
@czM1K3 yeah no problem at all, would suggest then a v0.1.0 release as [Semver](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) suggests using that number for the initial release
